mlocal/frags/Makefile.stub

```diff
@@ -226,6 +226,17 @@ check: $(BUILDDIR)/vendors-done
 	$(V)cd $(SOURCEDIR) && golint -set_exit_status `go list ./...`
 	@echo "       PASS"
 
+.PHONY: dist
+dist: $(BUILDDIR)/vendors-done
+	@echo " DIST setup VERSION"
+	$(V)git describe --match 'v[0-9]*' --dirty --always > $(SOURCEDIR)/VERSION
+	@echo " DIST create tarball"
+	$(V)cd $(SOURCEDIR) && $(V) mkdir tmpdir && \
```

Comment:
Only need `$(V)` once per series of commands. `$(V)cmd1 && cmd2 && cmd3` should silence all commands, since `@` is `makefile` syntax